tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/matchstmt/MatchStructuredErrorPatternsTest.java

```diff
@@ -199,4 +199,22 @@ public void testUnreachablePatterns() {
         BAssertUtil.validateError(resultNegative, ++i, unreachablePattern, 80, 13);
         BAssertUtil.validateError(resultNegative, ++i, unreachablePattern, 85, 13);
     }
+
+    @Test(description = "Test unsupported error match pattern")
+    public void testErrorMatchPatternNotSupportedErrors() {
+        CompileResult result = BCompileUtil.compile(
+                "test-src/statements/matchstmt/structured_error_match_patterns_negative2.bal");
+        int i = 0;
+        BAssertUtil.validateError(result, i++,
+                "error match pattern with reason as a 'constant' reference is currently not supported", 33, 15);
```

Comment:
tests needs to be updated for the error message change?

Reply:
Yes, I forgot that